frontend/src/tests/instructor-permission-test.js

```diff
@@ -46,6 +46,63 @@ export function instructorsPermissionTests(api) {
         expect(respSwitchBackUser).toHaveStatus("success");
     }
 
+    /**
+     * Updates a position to include instructor with <instructorId> and
+     * creates a DDAH for one of the assignments realted to that
+     * instructor.
+     *
+     * @param instructorId: int - the unique Id of an instructor
+     *
+     * @returns {Promise<Ddah>}
+     */
+    async function createDDAH(instructorId) {
+        // We first need to update position to include our instructor
+        const existingPosition = databaseSeeder.seededData.positions[0];
+        const updatedPosition = {
+            ...existingPosition,
+            instructor_ids: [...existingPosition.instructor_ids, instructorId],
+        };
+        const positionResponse = await apiPOST(
```

Comment:
Do a `let resp` here ad reuse the response variable, unless you have a reason you really need two responses hanging around at the same time.